src/pybind/mgr/cephadm/ssh.py

```diff
@@ -139,31 +139,47 @@ def remote_connection(self,
 
     async def _execute_command(self,
                                host: str,
-                               cmd: List[str],
+                               cmds: List[str],
```

Comment:
@phlogistonjohn in general I dont like appending the type on the name (specially when typing is used). In this case I prefer to rename the arg instead of all the instances. I renamed it to `cmd_components` .. not sure if it's the best name but at least it reflects what it's :)

Reply:
I can live with that name. The only risk of changing the argument's name would be if other code used the name (passing it as a keyword arg is the simplest case).

Reply:
@phlogistonjohn good point! I double checked just in case and no body was passing it as keyword arg :+1: